rust/operator-binary/src/crd/security.rs

```diff
@@ -293,42 +294,64 @@ impl KafkaTlsSecurity {
             args.push("-b".to_string());
             args.push(format!("localhost:{}", port));
             args.extend(Self::kcat_client_auth_ssl(Self::STACKABLE_TLS_KCAT_DIR));
+            args.push("-L".to_string());
         } else if self.has_kerberos_enabled() {
             let service_name = KafkaRole::Broker.kerberos_service_name();
+            let broker_port = match broker_listener_class.as_str() {
+                // for cluster-internal, kcat will connect using the broker name and the internal port
+                "cluster-internal" => port.to_string(),
```

Comment:
Why is it necessary, to make a difference between `cluster-internal` and the other Listener classes?

`/stackable/listener-broker/default-address/ports/kafka-tls` contains the internal port 9093 in case of `cluster-internal`.

Reply:
Ah, thanks for catching that: in my testing I didn't notice they were the same. See 0cd08a1

Reply:
In fact, I've ditched the log statement completely as we are not logging out anything that is not already in the pod yaml.

Reply:
Agreed; fwiw abstracting this logic away is half the point of using list-op.